status/format.go

```diff
@@ -66,26 +66,10 @@ func FormatSummary(taskRuns []*helpers.TaskRun) (string, error) {
 	return buf.String(), nil
 }
 
-// FormatCommentForFinishedPipelineRun builds a markdown comment for a list of finished integration TaskRuns
-func FormatCommentForFinishedPipelineRun(title string, results []*helpers.TaskRun) (string, error) {
```

Comment:
Is this no longer needed or we are formating comments different way?

Reply:
It's formatted in the same way without duplication caused by this function